db/schema.rb

```diff
@@ -93,6 +95,18 @@
     t.string   "general_feedback"
   end
 
+  create_table "matchings", force: :cascade do |t|
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.string   "name"
+    t.text     "projects"
+    t.text     "preferences"
+    t.text     "description"
+    t.text     "result"
+    t.string   "status",          default: "Collecting responses", null: false
+    t.text     "last_edit_users"
+  end
+
   create_table "orgs", force: :cascade do |t|
     t.string   "name"
     t.string   "description"
```

Comment:
Schema modified for two features: 
- add semester info to engagements
- new matching feature